### PR TITLE
Vlaicu getting locale

### PR DESCRIPTION
**Pull request by ltiseanu**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Documentation
- [ ] I have updated the sidekick documentation.
- [ ] I have updated new baselines in GhostInspector.

Test URLs:
- Before: https://main--www-landing-pages--bitdefender.hlx.page/drafts/test-page/
- After: https://vlaicu-locale--www-landing-pages--bitdefender.hlx.page/drafts/test-page/

### CODE REVIEW

File: _src-lp/scripts/vendor/product.js [hunk 1]
```diff
@@ -310,8 +311,17 @@ export class Locale {
         }
       }
 
-      // Construct the locale string
-      const locale = `${language}-${country}`;
+      // fetch locale
+      const fetchLocale = `https://www.bitdefender.com/p-api/v1/countries/${country.toUpperCase()}/locales`;
```
Comment (enake) [inline _src-lp/scripts/vendor/product.js]: Muta constanta la inceput de fisier. All caps.

File: _src-lp/scripts/vendor/product.js [hunk 2]
```diff
@@ -310,8 +311,17 @@ export class Locale {
...
+      fetch(fetchLocale).then(response => {
```
Comment (enake) [inline _src-lp/scripts/vendor/product.js]: Ai nevoie sa await dupa acel fetch.

File: _src-lp/scripts/vendor/product.js [hunk 3]
```diff
@@ -310,8 +311,17 @@ export class Locale {
...
+        if (!response.ok) throw new Error('err ' + response.statusText);
+        return response.json();
+      }).then(data => {
+        locale = data[0].locale;
```
Comment (enake) [inline _src-lp/scripts/vendor/product.js]: locale nu va avea nicio valoare

**Review by enake — CHANGES_REQUESTED** [review]

File: _src-lp/scripts/vendor/product.js (hunk 1 above)
Comment (ltiseanu) [reply _src-lp/scripts/vendor/product.js]: done

File: _src-lp/scripts/vendor/product.js (hunk 2 above)
Comment (ltiseanu) [reply _src-lp/scripts/vendor/product.js]: done

File: _src-lp/scripts/vendor/product.js (hunk 3 above)
Comment (ltiseanu) [reply _src-lp/scripts/vendor/product.js]: done

**Review by enake — APPROVED** [review]

**Review by inastase — APPROVED** [review]

**Pull request merged**